components/ai_chat/BUILD.gn

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2023 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+source_set("ai_chat") {
+  sources = [
+    "ai_chat_utils.cc",
```

Comment:
It seems to me that the api for communicating with the endpoint is shareable with ios and should therefore be part of the component and otherwise having the component seems pointless and the features should just go in browser/ai_chat

Reply:
Sorry to @nullhook for asking you to move everything back and forth. I suppose I misunderstood the pattern of when to make a component for the c++ - I thought we were moving to an approach where we put everything in brave/browser as much as possible unless it's needed by another component (or by iOS or something external). 
@bridiver In this case we did move everything to brave/browser, but not features because it's used by `components/sidebar` to selectively add the default sidebar entry for AIChat.
@nullhook I'll move everything that can be moved back in to components/ai_chat

Reply:
Everything is moved to `components/ai_chat` now. I didn't split to `components/ai_chat/content/` etc, we could do that at a later time if neccessary.